tests/cypress/tests-e2e/drawing.cy.js

```diff
@@ -891,6 +895,21 @@ describe('Drawing module tests', () => {
                 '[data-cy="drawing-toolbox-export-button"] [data-cy="dropdown-main-button"]'
             ).should('have.attr', 'disabled')
             cy.get('[data-cy="drawing-toolbox-share-button"]').should('have.attr', 'disabled')
+
+            //draws something new to verify that the kml id being sent is different
+            cy.clickDrawingTool(EditableFeatureTypes.LINEPOLYGON)
+            cy.get('[data-cy="ol-map"]').click(100, 250)
+            cy.get('[data-cy="ol-map"]').click(150, 250)
+            cy.get('[data-cy="ol-map"]').click(150, 280)
+
+            let newKmlId = null
+            cy.wait('@post-kml').then((interception) => {
```

Comment:
I would slightly rework the post-kml interception, because we have a collision risk (even if it's small, I would prefer to remove any form of flaky-ness).

Currently, it's a random 4-number number followed by file_id
We should ensure we can't get the same number twice in a row.